contracts/exchange-forwarder/contracts/src/MixinExchangeWrapper.sol

```diff
@@ -173,6 +190,15 @@ contract MixinExchangeWrapper is
                 remainingTakerAssetFillAmount
             );
 
+            // Account for the ERC20Bridge transfering more of the maker asset than expected.
+            if (makerAssetProxyId == erc20BridgeProxyId) {
+                uint256 balanceAfter = IERC20Token(tokenAddress).balanceOf(address(this));
+                makerAssetAcquiredAmount = LibSafeMath.max256(
```

Comment:
Why don't we just set `makerAssetAcquiredAmount` to `max(makerAssetAcquiredAmount, balanceAfter)`? Would that be so wrong?

Reply:
two edge cases: 
- someone accidentally transfers tokens to the Forwarder
- `makerAsset` is WETH, which seems silly but given the flexibility of the bridge pattern I could see there being a use case for this at some point 🤷‍♂